troposphere/validators/ssm.py

```diff
@@ -97,6 +97,8 @@ def operating_system(os):
     valid_os = [
         "AMAZON_LINUX",
```

Comment:
Might make sense to add missing `ALMA_LINUX` as well

```suggestion
        "ALMA_LINUX",
        "AMAZON_LINUX",
```